PF/ASSIGNMENT3/src/JobScheduler.java

```diff
@@ -0,0 +1,54 @@
+
+
+import java.util.Scanner;
+
+
+public class JobScheduler {
+	/**
+	 * @param jobs
+	 * @param numberOfJobs
+	 * function returning completion time for the given jobs
+	 */
+	public int[] completionTime(int jobs[][], int numberOfJobs){
+		int completionTime[]=new int[numberOfJobs];
+		int sum=0;
+		for(int i=0;i<numberOfJobs;i++){
+			sum=sum+jobs[i][1];
+			completionTime[i]=sum;//Completion time is sum of burst time
+		}
+		return completionTime;
+	}
+	
+	/**
+	 * @param jobs
+	 * @param numberOfJobs
+	 * @param completionTime
+	 * function returning turn around time
+	 */
+	public int[] turnAroundTime(int jobs[][],int numberOfJobs, int completionTime[]){
+		int turnAroundTime[]=new int[numberOfJobs];
+		for(int i=0;i<numberOfJobs;i++){
```

Comment:
for(int i=0;i<numberOfJobs;i++){

should have proper spacing 

for(int I = 0; I < numberOfJobs; i++) {